books.rb

```diff
@@ -0,0 +1,15 @@
+require_relative ' rental'
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Great job so far, as you did for title and author we also need getter and setter for rentals, kindly fix that.